clients/da-vinci-client/src/main/java/com/linkedin/davinci/serializer/avro/MapOrderPreservingDatumReader.java

```diff
@@ -18,15 +16,6 @@ public MapOrderPreservingDatumReader(Schema writer, Schema reader) {
     super(writer, reader);
   }
 
-  @Override
-  protected Object newArray(Object old, int size, Schema schema) {
```

Comment:
Do we still need this class at all?

Reply:
This class was kept in case the fast-avro adoption has some regression. 
After some stabilization period, we can completely remove these classes.